### DISCUSSION TITLE
Problems with bluetooth

### DISCUSSION THREAD

**Discussion by darts2020** [rern/rAudio#1028] [Bug Reports]
Problems with bluetooth
On version 20220916 I can not connect my smartphone via bluetooth. After the pair, the connection immediately breaks and a message appears:

Mixer not ready
Power it off > on / Reconnect again

And so constantly. Tried different smartphones and tablets - the same effect.

Version 20210813 was installed. Bluetooth worked fine. Then I installed the latest current version on the site from the image . And there was a problem that I described above. What do you advise?

**2023.09.02**: I used Cyrillic characters in the smartphone name. I removed these symbols and the bluetooth connection was restored. But on version 20220916 it was possible to use Cyrillic characters

**Comment by a-cormorant** [comment]
@darts2020 Heyo, I've been banging my head against this as well. This is what eventually worked for me on a Pixel 5.

Turn ON Bluetooth from the System menu.
Turn ON Discoverability in the Bluetooth settings menu.
Pair your device to rAudio. (It will pair and then disconnect.)
Turn OFF Discoverability.
Reconnect your device to rAudio.

This does not seem to work for connecting a Win 10 laptop to rAudio, and I haven't tried any other Android devices.

**Comment by newbee90** [comment]
I have the same problems with version 20230101
tried Discoverability on and off

**Comment by HJLK** [comment]
I have the a Bluetooth problem with version 20230218 (20230131 x64 updated) with connecting my BeoPlay Portal headphone.

It did work OK at first (with all the default Bluetooth settings), then I lost contact, for no reason it seemed. Connecting again was impossible, so I told Bluetooth to "forget" the headphone.  After a BT discovery the headphone was discovered, paired en connected, and within 5 seconds it was disconnected. After Connect I see "Ready", but the device is still disconnected (no green point). 
When I set Discovery OFF, connecting succeeds (green point), but no sound is heard... 
And after that **disconnect** and **forget** do not work. 
I will try to restart from the image again, and hope it works normal.

**2023-03-02:** After restarting from the image I see (and hear) that Bluetooth works ok now, also after updating to latest version. I notice the following:
1) mpd output is switched to Bluetooth, and my Hifiberry is soundless. I cannot switch back to Hifiberry via the GUI rAudio.local. When I switch my headphone off, my Hifiberry sounds again. Is this INTENTIONAL?
2) in the first minute (or so) the sound via Bluetooth is bad: it stops and starts again a number of times. After that the sound is good and stable. What might be the cause of this?

**2023-03-13:** The Bluetooth connection to my BeoPlay Portal headphone is now stable as for recognition and connection, but still in the first minute the sound quality is bad. If my Bluetooth headphone is connected my Hifiberry DAC+ is soundless, and the green led is OFF.
